### PR TITLE
feat:add alpha env variables for callback

### PR DESCRIPTION
**Pull request by StevenHee**
Need to set sign/out domain for alpha. Conditional test for alpha domain temporary so detection via `window.location` should suffice.

### CODE REVIEW

File: src/assets/environments/environment.json [hunk 1]
```diff
@@ -45,5 +45,28 @@
       "redirectSignIn": "https://stage-revalidation.tis.nhs.uk",
       "redirectSignOut": "https://stage-revalidation.tis.nhs.uk"
     }
+  },
+  "alpha": {
+    "production": true,
```
Comment (CaiWillis) [inline src/assets/environments/environment.json:50]: Everything else here seems to be stage/preprod, is this correct?

Comment (StevenHee) [reply src/assets/environments/environment.json:50]: > Everything else here seems to be stage/preprod, is this correct?

Yes, that's correct. Alpha is using all the backend services from preprod including cognito. The only real differences is once its authenticated, it returns to alpha FE

File: src/assets/environments/environment.json [hunk 2]
```diff
@@ -45,5 +45,28 @@
...
+    "dateFormat": "dd/MM/yyyy",
+    "awsConfig": {
+      "domain": "stage-auth.tis.nhs.uk",
+      "region": "eu-west-2",
+      "scope": ["openid", "aws.cognito.signin.user.admin"],
+      "authenticationFlowType": "USER_PASSWORD_AUTH",
+      "responseType": "token",
+      "userPoolId": "eu-west-2_o5Es6Bpl8",
+      "userPoolWebClientId": "r28c0u6jpc7scm9e2anhh274r",
+      "bucketName": "tis-revalidation-concerns-upload-preprod",
```
Comment (CaiWillis) [inline src/assets/environments/environment.json:66]: wrong bucket name?

Comment (StevenHee) [reply src/assets/environments/environment.json:66]: > wrong bucket name?

It's fine. The only distinction between alpha and preprod is the hosting of the FE

**Review by CaiWillis — APPROVED** [review]

**Pull request merged**